src/TokenWrapper.sol

```diff
@@ -79,21 +137,21 @@ contract TokenWrapper is ITokenWrapper, ERC20 {
     /**
      * @notice Wraps the underlying token `value` and mints wrapper tokens into `msg.sender`'s balance.
      * @dev The `totalSupply` of the wrapped token MUST be less than or equal to the underlying token balance of the current contract.
-     * @param value The value to be wrapped.
+     * @param value The `wei` value to be wrapped.
```

Comment:
Let's use "wad" instead of "wei". "wad" is a general concept in Maker.
https://github.com/makerdao/dss/wiki/Numerical-Ranges#notation